src/otx/core/model/base.py

```diff
@@ -38,12 +38,12 @@
 from otx.core.data.entity.tile import OTXTileBatchDataEntity
 from otx.core.exporter.native import OTXNativeModelExporter
 from otx.core.metrics import MetricInput, NullMetricCallable
-from otx.core.optimizer.callable import OptimizerCallableSupportHPO
+from otx.core.optimizer.callable import OptimizerCallableSupportPickle
```

Comment:
Why do you change this naming?

Reply:
It's because Adaptive batch size feature also uses this class because it almost use same method as HPO when trying batch size.

Reply:
Then how about OptimizerCallableSupportHPTuning? I think it can include HPO and adaptive batch size.

Reply:
I don't know why the renaming is necessary

Reply:
so you think OptimizerCallableSupportHPO also can include meaning of adaptive batch size, right?

Reply:
Why not? Is batch size not a hyperparameter?

Reply:
ok I'll revert it

Reply:
done